OJP/OJP_Places.xsd

```diff
@@ -203,10 +203,23 @@
 	</xs:group>
 	<xs:simpleType name="PlaceTypeEnumeration">
 		<xs:restriction base="xs:string">
-			<xs:enumeration value="stop"/>
+			<xs:enumeration value="stop">
+				<xs:annotation>
+					<xs:documentation>stop covers StopPlace and StopPoint.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
 			<xs:enumeration value="address"/>
 			<xs:enumeration value="poi"/>
-			<xs:enumeration value="coord"/>
+			<xs:enumeration value="coord">
+				<xs:annotation>
+					<xs:documentation>deprecated. Is replaced by geoPosition.</xs:documentation>
+				</xs:annotation>
+			</xs:enumeration>
+			<xs:enumeration value="location">
```

Comment:
Is an address a location ? I guess yes, but may then be explicit. If not "address" needs to be added

Reply:
@Aurige Where would we have to add it? In Transmodel? the problem is, that it all should e a PLACE not a LOCATION. An Address is a sort of PLACE: So I guess that what we are doing here, is ok. Or should we change something?

Reply:
My fault, nothing to add, _address_ is just above... I don't know how I missed it, sorry